.github/workflows/run_tests.yaml

```diff
@@ -1,49 +0,0 @@
-name: Relation Engine test and deploy
```

Comment:
why delete this file?

Reply:
Because I wanted to lug it around local/docker03 without pushing to kbase/relation_engine because that tells everyone it failed and I couldn't push it to n1mus/relation_engine with the workflow because my token wouldn't allow it. Is temporary